gradle/spotless.gradle

```diff
@@ -0,0 +1,17 @@
+apply plugin: 'com.diffplug.spotless'
+spotless {
+
+    if (projectType.javaProjects.contains(project)){
+        java {
+            bumpThisNumberIfACustomStepChanges(1)
+            // currently we do not use spotless here, but our own bash solution - but spotless would be:
+            //licenseHeaderFile rootProject.file('gradle/spotless.license')
+            importOrderFile   rootProject.file('gradle/spotless.importorder')
```

Comment:
Please read again:
```
currently we do not use spotless here, but our own bash solution - but spotless would be:
licenseHeaderFile rootProject.file('gradle/spotless.license')
```
We automatically apply our MIT headers via our bash script . This does handle it correctly and works well.
Spotless is also able to apply license header files automatically.
I just kept the example code inside, so it's easier when we want to switch (if ever). Also it describes that we have a bash solution here. So new team members will not accidently "enhance" the spotless setup.

Reply:
It this case please change the comment to:

~~~
// we currently do not use spotless to apply license headers. The license headers are added by our own bash solution:  `apply-headers.sh`.
// in case, we wanted to use spotless to apply license headers the next line needs to be uncommented.
// licenseHeaderFile rootProject.file('gradle/spotless.license')
~~~

Reply:
As discussed, we just remove the comment.